src/aui-layout/js/aui-layout-builder-remove-row.js

```diff
@@ -147,6 +149,26 @@ LayoutBuilderRemoveRow.prototype = {
     },
 
     /**
+     * Protected method to set enableRemoveRows atribute as false.
+     *
+     * @method _disableRemoveRows
+     * @protected
+     */
+    _disableRemoveRows: function() {
+        this.set('enableRemoveRows', false);
+    },
+
+    /**
+     * Protected method to set enableRemoveRows atribute as true.
+     *
+     * @method _enableRemoveRows
+     * @protected
+     */
+    _enableRemoveRows: function() {
```

Comment:
Yeah, Maira. I've understood. I'm trying to figure out how to recognise when the attribute value was changed by a user user wish or by the events. Do you have any advice?